src/main/java/com/cloudant/client/api/model/FindByIndexOptions.java

```diff
@@ -142,7 +148,13 @@ public Integer getReadQuorum() {
     }
 
     public String getUseIndex() {
-        return useIndex;
+        return useIndex.toString();
     }
 
+    /**
+     * For use in {@link Database#getFindByIndexBody(String, FindByIndexOptions)}
+     */
+    public JsonArray getUseIndexAsJsonArray() {

```

Comment:
After chatting over Slack, I decided on calling GSON's fromJson method to parse the JSON.  Fixed in aba42c9